Add files via upload

DCrescimone1 opened the pull request:
These files are provided to use the Green Energy Hub testbed on RIAPS. the communication protocol used are:
-Modbus for an interaction between the BBBs and the DSP
-MQTT for interaction between BBBs and user to send command and receive messages.

For more detail about the code read the two "README" of the folders

MMetelko commented:
I moved the application to the apps-ncsu folder.  I see that there is good documentation included as pdf file.  Could you please updated the README.md template that I put in this folder to reflect the project?  The purpose of this readme is to help others know what this project is and what they need to do to make things work.  Since you have the pdf, provide at least summary information and then point them to the documents.

DCrescimone1 commented:
Hi Mary,
I have modified the readme.md file, I hope that it is enough, let me kindly
know if I need to add something else.

Kindest regards

Il giorno lun 22 lug 2019 alle ore 20:44 Mary Metelko <
notifications@github.com> ha scritto:

> I moved the application to the apps-ncsu folder. I see that there is good
> documentation included as pdf file. Could you please updated the README.md
> template that I put in this folder to reflect the project? The purpose of
> this readme is to help others know what this project is and what they need
> to do to make things work. Since you have the pdf, provide at least summary
> information and then point them to the documents.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RIAPS/riaps-apps/pull/26?email_source=notifications&email_token=AL23VGAAJMC62LVM77KTX43QAX5XHA5CNFSM4IFV3NB2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2QZJMQ#issuecomment-513905842>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AL23VGAHGU2ICOJCZ3LWFTLQAX5XHANCNFSM4IFV3NBQ>
> .
>


-- 
*Danny Crescimone*
*MS student of Mechatronic Engineering*
*North Carolina State University*
*FREEDM Systems Center*

MMetelko commented:
Thank you for updating this.  Could you please take another editing pass on this information.

DCrescimone1 commented:
> Thank you for updating this. Could you please take another editing pass on this information.

I am sorry, but I didn't understand what I need to change now.

MMetelko commented:
I made edits to clarify information and remove work that was still in progress.  The only content change I made was the fourth installation instruction step.  The public key indicated is already in the authorized_keys file on the BBBs, but it was not on the VM.  This was fixed in v1.1.16.  You might want to update your instructions also (README_MQTTModbus_instructions.pdf).
